python_modules/libraries/dagster-k8s/dagster_k8s/job.py

```diff
@@ -653,6 +663,11 @@ def construct_dagster_k8s_job(
         "service_account_name", job_config.service_account_name
     )
 
+    scheduler_name = (
+        pod_spec_config.pop("scheduler_name", job_config.scheduler_name)
+        or job_config.scheduler_name
```

Comment:
we don't need both the second arg to pop and the or right?

Reply:
good catch